embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/payload/ExceptionInfo.kt

```diff
@@ -35,6 +35,10 @@ internal class ExceptionInfo internal constructor(
     @Json(name = "length")
     val originalLength: Int? = lines.size.takeIf { it > STACK_FRAME_LIMIT }
 
+    fun toNewPayload(): io.embrace.android.embracesdk.internal.payload.ExceptionInfo {
```

Comment:
nit: is the full import needed?

Reply:
Yeah it's needed to disambiguate right now. @priettt how about adding a prefix, like `LegacyExceptionInfo`, to affected classes? I'll roll that out across the other PRs tomorrow if you're happy with the approach